### ISSUE TITLE
Print warnings on use of "bad code"

### ISSUE THREAD

**Issue by imbrianj** [danbeam/oo-css#7]
Things such as the use of !important, exceeding the 4096 selector limit, obvious typos (possibly comparing HTML elements and CSS declaration names against white lists).  This is likely better place in a CSSLint program.

**Comment by danbeam** [comment]
Yeah, I was thinking about possibly making some type of lintiness to this, but we'll see.  There is a -v option in the compressor right now, but I'm not sure it works on CSS.
